<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>填充模式選擇</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>隨機ascii英數</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>混合語言</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>繁中姓名</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>簡中姓名</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>英文姓名</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>日文姓名</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>韓文姓名</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>檔案設定</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>411</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>輸入檔案</string>
     </property>
     <widget class="QLineEdit" name="InputFile_LineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>391</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="InputFile_Button">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>開啟檔案</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>21</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>檔案位置：</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>411</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>輸出檔案</string>
     </property>
     <widget class="QLineEdit" name="OutputFile_LineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>391</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="OutputFile_Button">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>開啟檔案</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>21</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>檔案位置：</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>431</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>執行</string>
    </property>
    <widget class="QPushButton" name="Start_Button">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>開始混淆</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>執行結果：</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="Result_TextBrowser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>391</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>491</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>填充預覽</string>
    </property>
    <widget class="QTextBrowser" name="PreView_TextBrowser">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>21</y>
       <width>481</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
